starlite/middleware/exceptions/debug_response.py

```diff
@@ -0,0 +1,121 @@
+import inspect
+from html import escape
+from pathlib import Path
+from traceback import format_exception
+from typing import TYPE_CHECKING, List
+
+from starlite.enums import MediaType
+from starlite.response import Response
+from starlite.status_codes import HTTP_500_INTERNAL_SERVER_ERROR
+
+if TYPE_CHECKING:
+    from starlite.connection import Request
+
+tpl_dir = Path(__file__).parent / "templates"
+
+
+def get_symbol_name(frame: inspect.FrameInfo) -> str:
+    """Return full name of the function that is being executed by the given
+    frame."""
+    classname = ""
+    locals_dict = frame.frame.f_locals
+    # probably we are inside of an instance method
+    # this piece assumes that the code uses standard names "self" and "cls"
+    # in instance and class methods
+    if locals_dict.get("self"):
+        classname = f"{type(locals_dict['self']).__name__}."
+    # inside a class method?
+    elif locals_dict.get("cls"):
+        classname = f"{locals_dict['cls'].__name__}."
+
+    return f"{classname}{frame.function}"
+
+
+def create_line_html(
+    line: str,
+    line_no: int,
+    frame_index: int,
+    idx: int,
+) -> str:
+    """Return html representation of a line including real line number in the
```

Comment:
Docstrings aren't consistent with the rest of the project.

Reply:
by which @peterschutt means we should use google style doc strings and make sure all params are documented properly.

Reply:
@Goldziher @peterschutt can we clarify requirements for documenting functions and methods because I am confusedl

CONTRIBUING.md states

> All public functions and methods should be documented with a doc string. The project uses the [Google style docstring](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html). Private methods should have a doc string explaining what they do, but do not require an elaborate doc string.

I guess we should update the doc if we want to follow Google style at all times. 

Also, I saw a lot of code that uses simple docstrings. Like [this](https://github.com/starlite-api/starlite/blob/main/starlite/plugins/tortoise_orm.py#L102). Am I right assuming that it is due to it being an old piece?

Reply:
> @Goldziher @peterschutt can we clarify requirements for documenting functions and methods, please, because I am confused
> 
> CONTRIBUING.md states
> 
> > All public functions and methods should be documented with a doc string. The project uses the [Google style docstring](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html). Private methods should have a doc string explaining what they do, but do not require an elaborate doc string.
> 
> I guess we should update the doc if we want to follow Google style at all times.
> 
> Also, I saw a lot of code that uses simple docstrings. Like [this](https://github.com/starlite-api/starlite/blob/main/starlite/plugins/tortoise_orm.py#L102). Am I right assuming that it is due to it being an old piece?

Yes, its an ongoing effort - but all new code that has public methods should be fully documented.

Reply:
Understood. Updated doc strings.

Reply:
@Goldziher @peterschutt could you please check docstrings again. English is not my first language so I would appreciate your help here.